04-custom-asynchronous-sequences-with-asyncstream/projects/starter/Blabber.xcodeproj/project.pbxproj

```diff
@@ -278,6 +278,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		795EF16326E4D0C20074D4C2 /* ShellScript */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
@jrturton what is this setting 🤔

Reply:
Without it the swiftlint phase causes a compiler warning. It’s new in xcode 14

Reply:
Oh, gotcha yes, I see ...

Reply:
Thanks